install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -120,6 +120,13 @@ rules:
   - delete
   - list
   - watch
```

Comment:
I see this PR is marked for backports into v1.10 but the above permissions (seem related to current PR) don't exist in v1.10. Is that expected? Do we need to backport something else?

Reply:
So this part of the PR would be unnecessary. Because [this permission](https://github.com/cilium/cilium/blob/v1.10/install/kubernetes/cilium/templates/cilium-operator-clusterrole.yaml#L125) gives cilium-operator the permissions it needs and [this permission](https://github.com/cilium/cilium/blob/v1.10/install/kubernetes/cilium/templates/cilium-agent-clusterrole.yaml#L86) gives cilium-agent everything it needs. We reworked them in v1.12 to be more specific.

Reply:
Oh, I see. Thanks for the explanation!